Slim/Plugin/SpotifyLogi/Plugin.pm

```diff
@@ -86,7 +86,9 @@ sub initPlugin {
 	);
 	
 	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
-	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
```

Comment:
How do I limit the pull request to only certain files? I assumed that the
GIT system only merged changed files...

Magnar.


2016-08-30 20:25 GMT+02:00 michaelherger <notifications@github.com>:

> In Slim/Plugin/SpotifyLogi/Plugin.pm
> <https://github.com/Logitech/slimserver/pull/110#discussion_r76851584>:
>
> > @@ -86,7 +86,9 @@ sub initPlugin {
> >  	);
> >  	
> >  	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
> > -	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
>
> This shouldn't be part of your pull request... Make sure only the text
> files (and others you might have changed) are part of it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Logitech/slimserver/pull/110/files/3b71770242024bf1d6354065b1cec89c832b781c#r76851584>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQYZHth4tGgNWj0N8MafgppbSaQruSejks5qlHWJgaJpZM4JwxYj>
> .
>